apps/site/src/components/pages/dashboard/utils.ts

```diff
@@ -49,29 +49,29 @@ export const getDashboardSectionCards = (props: {
         },
       },
       {
-        title: "Publish a block",
+        title: "My blocks",
         colorGradient:
           "linear-gradient(310.17deg, #FFB172 -167.67%, #9482FF 13.54%, #84E6FF 126.83%)",
         colorGradientOnHover:
           "linear-gradient(304.41deg, #FFB172 -167.57%, #9482FF -22.66%, #84E6FF 53.07%)",
-        description: "Build a block you’re ready to release on the Hub?",
+        description: "Browse your blocks or publish a new one to the Þ Hub",
         link: {
-          title: "Publish a block",
+          title: "My blocks",
           href: shouldAllowNpmBlockPublishing
             ? "/blocks/publish"
             : `${props.profileLink}/blocks`,
```

Comment:
This `href` seems incorrect

Reply:
It's not used - it relates to a flow that is basically hidden and inaccessible right now, as I understand @CiaranMn? I also didn't introduce this.

Reply:
I understand, but if we ever did enable the flow this would link to the incorrect place. I think we should just change it to
```suggestion
          href: `${props.profileLink}/blocks`,
```

Reply:
I'll Slack you re: this